pkg/cloudprovider/providers/mesos/config.go

```diff
@@ -35,17 +37,30 @@ const (
 //  mesos-master        = leader.mesos:5050
 //	http-client-timeout = 500ms
 //	state-cache-ttl     = 1h
+//
+// [mesos-node]
+//  labels = {"rack": "a", "gen": "2014"}
 
-type ConfigWrapper struct {
-	Mesos_Cloud Config
+type Config struct {
+	Mesos_Cloud Cloud
```

Comment:
I believe that I read in the gcfg docs that this is mandatory. But maybe I am wrong.